geoips/plugins/modules/readers/abi_netcdf.py

```diff
@@ -54,17 +55,17 @@
 family = "standard"
 name = "abi_netcdf"
 
-log = logging.getLogger(__name__)
 
 nprocs = 6
 
 try:
     ne.set_num_threads(nprocs)
 except Exception:
-    print(
-        f"Failed numexpr.set_num_threads in {__file__}. "
-        f"If numexpr is not installed and you need it, install it."
-    )
+    log = logging.getLogger(__name__)
+(
+    f"Failed numexpr.set_num_threads in {__file__}. "
+    f"If numexpr is not installed and you need it, install it."
+)
```

Comment:
This should have a log statement, shouldn't it? This looks like it won't actually do anything.

Reply:
I think there must have been a bad copy/paste or search and replace.  Updated to where I think it should be...